data/brands/amenity/restaurant.json

```
@@ -3836,6 +3836,17 @@
         "name": "Poivre Rouge"
       }
     },
+    {
+      "displayName": "Pomodoro",
+      "locationSet": {"include": ["es"]},
+      "tags": {
+        "amenity": "restaurant",
+        "brand": "Pomodoro",
+        "brand:wikidata": "Q7316076",
+        "cuisine": ["pasta","pizza","burger","italian"],
```

```suggestion
        "cuisine": "pasta;pizza;burger;italian",
```

https://github.com/osmlab/name-suggestion-index/wiki#about-openstreetmap

I don't know why the test even passed, but looks like current schema haven't change
So, the value of `cuisine` key shouldn't be a **list** of string, a **string** is enough.

(If I missed some technical changes, please point it to me, thanks!)